session/session.go

```diff
@@ -165,13 +165,15 @@ func (s *Session) Run() error {
 	// In some cases if you set LD_LIBRARY_PATH it is ignored.
 	// This is disappointing to say the least. We just bind a few things into /
 	// bind *may* hide local resources but for now it's the least worst option.
-	if tab, ok := os.LookupEnv("CPU_FSTAB"); ok {
-		verbose("Mounting %q", tab)
-		if err := mount.Mount(tab); err != nil {
-			verbose("fstab mount failure: %v", err)
-			// Should we die if the mounts fail? For now, we think not;
-			// the user may be able to debug. Just record that it failed.
-			s.fail = true
+	for _, fstab := range []string{"CPU_FSTAB"} {
```

Comment:
do we need looping for one element or something is missing?

Reply:
I think I'll leave it as a loop if that is OK? At some point, we needed it, and might again?